internal/provider/resources/block.go

```diff
@@ -74,9 +74,11 @@ func (r *BlockResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 			"`block` resources represent configurations for schemas for all different Block types. " +
 			"Because of the polymorphic nature of Blocks, you should utilize the `prefect` [CLI](https://docs.prefect.io/latest/getting-started/installation/) to inspect all Block types and schemas." +
 			"\n" +
-			"Use `prefect block types ls` to view all available Block type slugs, which is used in the `type_slug` attribute." +
+			"*Note:* you should be on version `3.0.0rc1` or later to use the following commands:" +
```

Comment:
I guess technically `3.0.0rc1` is only required for the "data schema for a given Block type" portion below, and isn't really a requirement for just `prefect block type ls` - but that might be splitting hairs.

Reply:
hahah right. it started to become a mouthful, so i just glossed over some of the details